tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -130,42 +142,178 @@ private static ArgumentParser argParser() {
                 .metavar("THROUGHPUT")
                 .help("throttle maximum message throughput to *approximately* THROUGHPUT messages/sec");
 
+        parser.addArgument("--num-threads")
+                .action(store())
+                .required(true)
+                .type(Integer.class)
+                .metavar("NUM-THREADS")
+                .dest("numThreads")
+                .help("The value bound of the random integers in message payload to simulate different compression ratio.");
```

Comment:
Is there any reason that users may want to change value for this configuration? If yes, can this API doc provide users some clue as to how this configuration affects the compression ratio? E.g., higher value for this config should provide lower compression ratio. With value 1, the compression ratio is xxx. With value MAX_INT, the compression ratio is xxx.

BTW, any reason we select a random int for every four byte instead of random byte for every byte?